gensrc/script/vectorized/vectorized_functions.py

```diff
@@ -471,6 +471,7 @@
     [90500, 'bitmap_to_array', 'ARRAY_BIGINT', ['BITMAP'], 'BitmapFunctions::bitmap_to_array', False],
     [90600, 'bitmap_max', 'BIGINT', ['BITMAP'], 'BitmapFunctions::bitmap_max', False],
     [90700, 'bitmap_min', 'BIGINT', ['BITMAP'], 'BitmapFunctions::bitmap_min', False],
+    [90800, 'base64_to_bitmap', 'BITMAP', ['VARCHAR'], 'BitmapFunctions::base64_to_bitmap', False],
```

Comment:
what does base64_to_bitmap mean? What's the format or standard of the base64 content?

Reply:
1、in client(java or cpp), we use BitmapValue, and serialize、base64 encoded, base64string is the input
2、base64 encode algorithm is the same
I add the java demo in the comment